kudos-services/src/main/java/org/exoplatform/kudos/listener/analytics/KudosSentListener.java

```diff
@@ -116,7 +118,7 @@ private void addEventStatistic(Kudos kudos) {
 
     statisticData.setModule("social");
     statisticData.setSubModule("kudos");
-    statisticData.setOperation("sendKudos");
+    statisticData.setOperation("sendKudosFrom" + source);
     statisticData.setUserId(Long.parseLong(kudos.getSenderIdentityId()));
     statisticData.addParameter("activityId", activityId);
```

Comment:
For good statistic collection, it's better to keep the operation as it is and add a dedicated field for the "Source" information

```suggestion
    statisticData.setOperation("sendKudos");
    statisticData.setUserId(Long.parseLong(kudos.getSenderIdentityId()));
    statisticData.addParameter("activityId", activityId);
    statisticData.addParameter("uiSource", source);
```

Reply:
or you can name it `kudosEntityType`